src/Workspaces/CSharp/Portable/Workspace/LanguageServices/CSharpCompilationFactoryService.cs

```diff
@@ -45,11 +45,6 @@ CompilationOptions ICompilationFactoryService.GetDefaultCompilationOptions()
 
         GeneratorDriver? ICompilationFactoryService.CreateGeneratorDriver(ParseOptions parseOptions, ImmutableArray<ISourceGenerator> generators, AnalyzerConfigOptionsProvider optionsProvider, ImmutableArray<AdditionalText> additionalTexts)
         {
-            // https://github.com/dotnet/roslyn/issues/42565: for now we gate behind langver == preview. We'll remove this before final shipping, as the feature is langver agnostic
```

Comment:
Can you also update this line?

https://github.com/dotnet/roslyn/blob/9737cff23a0064b6b070bee3d4bc356ff49925ae/src/Workspaces/CoreTest/SolutionTests/SolutionWithSourceGeneratorTests.cs#L37

(just delete the parseOptions parameter entirely)